lib/computer.rb

```diff
@@ -22,7 +22,7 @@ def best_move
     board.free_positions.each do |position|
       board.set_mark(mark, position)
       result = alphabeta(mark, -1000, 1000)
-      board.remove_mark(position)
+      board.remove_mark(position, position)
```

Comment:
You maed me realise that after some refactoring set_mark and remove_mark are a duplication now. 

I set the position twice because the second argument  will be the content of the mark, and I want to have the id of the position.

May be remove_mark should take just one argument and use this argument for find the position and set the value